app/controllers/issues_controller.rb

```diff
@@ -22,7 +23,7 @@ def new
   end
 
   def show
-    @issue = Issue.find(params[:id])
+    @voted = @issue.votes.where(user_id: current_user.id)
     load_relevant_issues
     redirect_back(fallback_location: root_path) unless \
                                                 @issue.can_read?(current_user)
```

Comment:
Something might be changed in rails 5 but
```ruby @voted = @issue.votes.where(user_id: current_user.id)``` returns arel instance not ```Vote``` instance
so you can't call ```ruby @issue.can_read?(current_user)```